requirements/base.txt

```diff
@@ -4,13 +4,14 @@
 #
 #    make upgrade
 #
+asgiref==3.2.4            # via django
 certifi==2019.11.28       # via requests
 chardet==3.0.4            # via requests
 django-waffle==0.20.0     # via edx-django-utils
-django==2.2.11            # via edx-django-utils
+django==3.0.4             # via edx-django-utils
```

Comment:
Do we really want dev environments to get Django 3 by default already?

Reply:
Tox is still running for 111 only right now I can add a new env for this? to test both.